app/konnect/network-resiliency.md

```diff
@@ -4,46 +4,47 @@ toc: false
+and data plane nodes must communicate with each other to receive and send 
+configurations. If communication is interrupted and either side can't send or 
+receive config, data plane nodes can still continue proxying traffic to clients.
 
 Whenever a data plane node receives new configuration from the control plane,
 it immediately loads that config into memory. At the same time, it caches
 the config as `config.json.gz` into `/usr/local/kong` by default.
 
 ## Communication
 
-### How do the control plane and data planes communicate?
+### How do the control plane and data plane nodes communicate?
 
-Data traveling between control planes and data planes is secured through a
+Data traveling between control planes and data plane nodes is secured through a
 mutual TLS handshake. 
-Data planes initiate the connection to the {{site.konnect_short_name}} control plane.
-Once the connection is established, the control plane can send configuration data to the connected data planes.
+Data plane nodes initiate the connection to the {{site.konnect_short_name}} control plane.
+Once the connection is established, the control plane can send configuration data to the 
+connected data plane nodes.
 
-Normally, the data plane maintains a persistent connection with the control
-plane. The data plane sends a heartbeat to the control plane every 30 seconds to
+Normally, each data plane node maintains a persistent connection with the control
+plane. The node sends a heartbeat to the control plane every 30 seconds to
 keep the connection alive. If it receives no answer, it tries to reconnect to the
 control plane after a 5-10 second delay.
 
-### What types of data travel between the {{site.konnect_saas}} control plane and the data planes, and how?
+### What types of data travel between the {{site.konnect_saas}} control plane and the data plane nodes, and how?
 
 There are two types of data that travel between the planes: configuration
 and telemetry. Both use the secure TCP port `443`.
 
-* **Configuration:** the control plane sends configuration data to any connected
-  data planes in the cluster.
+* **Configuration:** The control plane sends configuration data to any connected
+  data plane nodes in the cluster.
 
-* **Telemetry:** data plane nodes send usage information to the control plane
+* **Telemetry:** Data plane nodes send usage information to the control plane
   for Analytics and for account billing. Analytics tracks aggregate traffic by
   service, route, and the consuming application. For billing, Kong tracks the
   number of services, API calls, and active dev portals.
 
 Telemetry data does not include any customer information or any data processed
 by the data plane. All telemetry data is encrypted using mTLS.
 
-### How frequently does data travel between the control plane and data planes?
+### How frequently does data travel between the control plane and data plane nodes?
```

Comment:
```suggestion
### How frequently does data travel between the Konnect control plane and data plane nodes?
```

It feels like we have to be specific here. I think this coiuld either be "between the control plane and the data planes", or "....Konnect control plane and the data plane nodes".